package.json

```diff
@@ -60,7 +60,7 @@
     "example:android": "cd ./RNFBSDKExample && yarn android --no-jetifier",
     "example:clean": "cd RNFBSDKExample && \\rm -fr yarn.lock node_modules ios/Podfile.lock && cd ..",
     "example:install": "cd RNFBSDKExample && yarn && cd ios && (rm -f Podfile.lock && pod install || true) && cd ../..",
-    "example:devcopy": "yarn prepare && cp -rv android ios lib src *.podspec RNFBSDKExample/node_modules/react-native-fbsdk-next/",
+    "example:devcopy": "yarn prepare && cp -rv android ios lib src package.json *.podspec RNFBSDKExample/node_modules/react-native-fbsdk-next/",
```

Comment:
As mentioned in #168, this will fix the wrong file path pointing from `node_modules/react-native-fbsdk-next/package.json`